### PR TITLE
[DS-4404] Upgrade Mets crosswalk for oai to xslt 2.0 

### PR DESCRIPTION
**Pull request by anis-moubarik**
This removes the exslt functions and uses xslt 2.0 native ones for dates and times. This is not compatible with Xalan-J, so needs some consideration for backporting.

Related: https://github.com/DSpace/xoai/pull/68

### CODE REVIEW

**Comment by tdonohue** [comment]
@paulo-graca : Would you be interested in testing this PR briefly to be certain it doesn't cause any issues? It looks correct to me, but could use a second pair of eyes.  As you can see, it's a follow-up to the #68 PR that you tested previously.

**Comment by tdonohue** [comment]
As this is REQUIRED by [XOAI v3.2.11](https://github.com/DSpace/xoai/releases/tag/xoai-3.2.11), and that XOAI release is required #2592, I am flagging this as high priority for 7.0.

**Review by paulo-graca — APPROVED** [review]
The change looks ok to me, thank you @anis-moubarik  for adding this fix!

**Review by tdonohue — APPROVED** [review]
👍 Looks good to me too, and I trust @paulo-graca 's tests.  Merging.

**Comment by tdonohue** [comment]
After merging this, I'm seeing new errors on `master`.  See https://jira.lyrasis.org/browse/DS-4404?focusedCommentId=64173&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-64173

I suspect this was not a complete fix & needs further work.

**Pull request merged**
